test/jdk/jdk/incubator/concurrent/ScopedValue/ScopeValueAPI.java

```diff
@@ -291,15 +298,16 @@ public void testRebinding(ThreadFactory factory) throws Exception {
     /**
      * Test rebinding from null vaue to another value.
      */
-    @Test(dataProvider = "factories")
-    public void testRebindingFromNull(ThreadFactory factory) throws Exception {
+    @ParameterizedTest
+    @MethodSource("factories")
+    void testRebindingFromNull(ThreadFactory factory) throws Exception {
         test(factory, () -> {
             ScopedValue<String> name = ScopedValue.newInstance();
 
             // run
             ScopedValue.where(name, null, () -> {
                 assertTrue(name.isBound());
-                assertEquals(name.get(), null);
+                assertTrue(name.get() == null);
```

Comment:
Perhaps `assertNull(name.get())`?

Reply:
> Perhaps `assertNull(name.get())`?. Same on line 325.

Okay, we can do that, note that it impacts 45+ usages and I had hoped to avoid changing the tests too much.

Reply:
> note that it impacts 45+ usages and I had hoped to avoid changing the tests too much.

I hadn't realized this construct was used in multiple other places. I see that you already fixed them in a newer commit in this PR; thank you.